src/main/java/org/codehaus/mojo/license/api/DefaultThirdPartyTool.java

```diff
@@ -552,7 +554,7 @@ public SortedProperties loadUnsafeMapping( LicenseMap licenseMap,
             MavenProject project = artifactCache.get( id );
             if ( project == null )
             {
-                LOG.warn( "dependency [{}] does not exist in project.", id );
+                handleUnkownDependency(unkownFileRemedy,  "dependency [" + id + "] does not exist in project." );
```

Comment:
Probably cleaner to hand `id` down to `handleUnkownDependency` instead of passing the log message around.